projects/canopy/src/lib/hero-img/hero-img-card-subtitle/hero-img-card-subtitle.component.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../../styles/mixins.scss';
+
+.lg-hero-img-card-subtitle__heading {
+  > h1,
+  > h2,
+  > h3,
+  > h4,
+  > h5,
+  > h6 {
+    @include lg-font-size('3');
+
+    color: var(--color-white);
```

Comment:
Maybe this should be a more generic variable in case someone wants to override it to have the colour of the text different e.g. `--hero-img-color`.
Then in the scss module you would do:
```
...
  --hero-img-color: var(--color-white);
} 
```